NMDSdevopsServiceAdm/DataEngineering
# Remove purge workplaces

# Description
https://trello.com/c/UPf8ompH/281-epic-5-remove-purge-workplaces-must

# How to test
https://eu-west-2.console.aws.amazon.com/gluestudio/home?region=eu-west-2#/editor/job/remove-purge-workplaces-clean_ascwds_workplace_data_job/runs - job passes in pipeline

Need particular attention on the following tests, as I've change them to make them pass but I don't know what they are trying to do:
- test_does_not_use_org_children_with_different_import_dates - the test isn't adding more data that has the parent tag, or in anyway related to other data, so not sure what this is trying to achieve
- test_adds_correct_value_for_parent_workplaces - I've changed it for only one value, but I don't understand the significance of why being a parent influences the 24 month time window, so need to make sure this test is still behaving as expected.

# Developer checklist
- [x] Successful run in pipeline
